src/components/safe-apps/SafeAppsErrorBoundary/SafeAppsLoadError.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from 'next/router'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+import Button from '@mui/material/Button'
+import OpenInNew from '@mui/icons-material/OpenInNew'
+import { SUPPORT_CHAT_URL } from '@/config/constants'
+
+import css from './styles.module.css'
+
+const SafeAppsLoadError = (): React.ReactElement => {
+  const router = useRouter()
```

Comment:
The component that renders this component already has a router object, I'd pass it as a prop to avoid unnecessary function call. Or even pass the onClick handler

Reply:
done